orcid-model/src/main/java/org/orcid/jaxb/model/common_rc4/CreditName.java

```diff
@@ -53,6 +55,7 @@
 @XmlAccessorType(XmlAccessType.FIELD)
 @XmlType( propOrder = { "content" })
 @XmlRootElement(name = "credit-name")
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Let's be explicit. https://fasterxml.github.io/jackson-annotations/javadoc/2.6/com/fasterxml/jackson/annotation/JsonIgnoreProperties.html